src/turnkeyml/build/export.py

```diff
@@ -279,9 +280,31 @@ def fire(self, state: build.State):
         default_warnings = warnings.showwarning
         warnings.showwarning = _warn_to_stdout
 
+        stats = fs.Stats(
+            state.cache_dir, state.config.build_name, state.evaluation_id
+        )
+
+        # Verify if the exported model matches the input torch model
+        try:
+            # The `torch.onnx.verification.find_mismatch()` takes input arguments to the
+            # model as `input_args (Tuple[Any, ...])`
+            export_verification = torch.onnx.verification.find_mismatch(
+                state.model,
+                tuple(state.inputs.values()),
```

Comment:
How data-dependent is this? What happens if my inputs are all 0's? It would be very hard to detect a problem in that case.

Reply:
All examples use random data. There are no mentions of risks involved with using all zeros. But I would guess using 0s might cause this API to return `True` while there is a mismatch.